### ISSUE TITLE
org.springframework.data.relational.core.mapping '@Table' name property

### ISSUE THREAD

**Issue by kimdg1105** [infobip/infobip-spring-data-querydsl#72]
Hi. I use spring data jdbc and 'infobip-spring-data-querydsl:7.2.0' (also, infobip-spring-data-jdbc-annotation-processor.)

When i generate Q class, most of entities are created well. But, Entities with @Table( name = "somthing") annotation, Q class's constructor generated like this.

    public QSlotGroup(String variable) {
        super(SlotGroup.class, forVariable(variable), null, "");
        addMetadata();
    }

As you see, table name Field fill with "".

    public RelationalPathBase(Class<? extends T> type, PathMetadata metadata, String schema,
            String table) {
        super(type, metadata);
        this.schema = schema;
        this.table = table;
        this.schemaAndTable = new SchemaAndTable(schema, table);
    }

`@Table(name = "slot_group") -> @Table("slot_group")`

    public QSlotGroup(String variable) {
        super(SlotGroup.class, forVariable(variable), null, "slot_group");
        addMetadata();
    }

I solved it by changing it like this, but I think there may be other solutions or improvements, so I'm writing this issue.

**Comment by lpandzic** [comment]
Hm, something is missing here.
I've just pushed an annotation processor test that confirms that simple name work here - https://github.com/infobip/infobip-spring-data-querydsl/commit/b8b4aa13c13f0e2112ecda8015fb1e9b704ddf30

**Comment by lpandzic** [comment]
Are you using custom NamingStrategy?

**Comment by kimdg1105** [comment]
@lpandzic No, i'm not use custom NamingStrategy.

In spring data jdbc java doc, Annotation Type 'Table' has two Optional Elements, 'name' and 'value'.
those two elements are the same description "The mapping table name."

```
@Table(name = "slot_group") -> @Table("slot_group")
@Table(name = "slot_group") -> @Table(value = "slot_group")
```
using @Table like this, it works well. But when add 'name' element , it doesn't work.

**Comment by lpandzic** [comment]
Fixed in 7.2.4
